someip2val/src/lib/broker_feeder/actuator_subscriber.cc

```diff
@@ -0,0 +1,173 @@
+/********************************************************************************
+* Copyright (c) 2022 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+* information regarding copyright ownership.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License 2.0 which is available at
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* SPDX-License-Identifier: Apache-2.0
+********************************************************************************/
+#include <sstream>
+#include <thread>
+
+#include "simple_log.h"
+#include "actuator_subscriber.h"
+
+// declare logger in root namespace
+LOGGER_STATIC_INIT("# ActuatorSubscriber::");
+
+namespace sdv {
+namespace broker_feeder {
+namespace kuksa {
+
+// helper for printing vector elements
+template <typename T> std::ostream& operator<<(std::ostream& os, const std::vector<T> &v) {
+    for (auto t : v) {
+        os << t << " ";
+    }
+    return os;
+}
+
+ActuatorSubscriber::ActuatorSubscriber(std::shared_ptr<CollectorClient> client) :
+    client_(client),
+    subscriber_active_(false),
+    cb_(nullptr),
+    actuators_()
+{
+    LOGGER_SET_LEVEL_ENV("KUKSA_DEBUG", LEVEL_INF);
+    // subscriber_context_ = client_->createClientContext();
```

Comment:
Remove commented code unless we see a need to keep. If so maybe add a comment on why we keep it.